students/HarveyHerela/lesson08/Circle.py

```diff
@@ -0,0 +1,99 @@
+from math import pi
+from functools import total_ordering
+
+import random
+import time
+
+@total_ordering
+class Circle:
+    """Basic Circle class. Supports:
+    Compute the circle’s area
+    Print the circle and get something nice
+    Be able to add two circles together
+    Be able to compare two circles to see which is bigger
+    Be able to compare to see if there are equal
+    (follows from above) be able to put them in a list and sort them
+    """
+    
+    def __init__(self, the_radius):
+        self._radius = the_radius
+        self._diameter = the_radius * 2
```

Comment:
take a look at this:
```
c = Circle(5)
print(c.radius)  # --> 5
c._diameter = 4
print(c.diameter) # --> 4
print(c.radius) # --> 5.. oops not good!
```
whenever I use setters/properties, I use double leading underscores -> `self.__diameter` this prevents the user from modifying vars directly, so they will always go via setters and this will ensure that proper logic happens.